packages/medusa/src/services/__tests__/cart.js

```diff
@@ -262,7 +276,7 @@ describe("CartService", () => {
       expect(cartRepository.save).toHaveBeenCalledTimes(1)
     })
 
-    it("creates a cart with a prefilled shipping address", async () => {
+    it("creates a cart with a prefilled shipping address but a country not part of the region", async () => {
```

Comment:
question: We should not be able to create a cart with a country not part of region. Is this incorrect naming of the test?

Reply:
Should throw `Shipping country not in region`

Reply:
yes the name should include the trow well spotted :)